Modules/Microsoft365DSC/Modules/M365DSCPermissions.psm1

```diff
@@ -1214,6 +1278,10 @@ Update-M365DSCAzureAdApplication -ApplicationName 'Microsoft365DSC' -Permissions
 .Example
 Update-M365DSCAzureAdApplication -ApplicationName 'Microsoft365DSC' -Permissions @(@{Api='SharePoint';PermissionName='Sites.FullControl.All'},@{Api='Graph';PermissionName='Group.ReadWrite.All'},@{Api='Exchange';PermissionName='Exchange.ManageAsApp'}) -AdminConsent -Credential $creds -Type Certificate -CertificatePath c:\Temp\M365DSC.cer
 
+.Example
+Update-M365DSCAzureAdApplication -ApplicationName $Microsoft365DSC -Permissions $(Get-M365DSCCompiledPermissionList -ResourceNameList Get-M365DSCAllResources -PermissionType Application -AccessType Read) -Type Certificate -CreateSelfSignedCertificate -AdminConsent -MonthsValid 12 -Credential $creds -CertificatePath c:\Temp\M365DSC.cer
+
+
```

Comment:
```suggestion
.Example
This will use credential $cred to create/update the application using an existing certificate.

Update-M365DSCAzureAdApplication -ApplicationName $Microsoft365DSC `
-Permissions $(Get-M365DSCCompiledPermissionList -ResourceNameList Get-M365DSCAllResources -PermissionType Application -AccessType Read) `
-Type Certificate `
-AdminConsent `
-Credential $creds `
-CertificatePath c:\Temp\Existing_cert.cer

.Example
This will use the existing certificate thumbprint (installed on the machine) to connect to an existing application, update permissions and roll over the auth certificate

Update-M365DSCAzureAdApplication -ApplicationName $Microsoft365DSC `
-Permissions $(Get-M365DSCCompiledPermissionList -ResourceNameList Get-M365DSCAllResources -PermissionType Application -AccessType Read) `
-Type Certificate `
-TenantId your_tenant.onmicrosoft.com `
-CertificateThumbprint <thumbprint of the existing M365DSC app auth certificate installed on the machine> `
-ApplicationId <AppID of an existing M365DSC app registration> `
-AdminConsent `
-CertificatePath c:\Temp\New_cert.cer `


```